lib/ui/widgets/bottom_bar.dart

```diff
@@ -28,6 +28,7 @@ class BottomBar extends StatelessWidget {
   Widget build(BuildContext context) {
     return BottomNavigationBar(
       type: BottomNavigationBarType.fixed,
+      elevation: 0,
```

Comment:
Just a question but why is this necessary?

Reply:
For the light theme it creates an ugly seperation under the BottomBar

<img src="https://user-images.githubusercontent.com/45638733/126871280-b81068ab-378b-4ed9-9fe2-ffce7fa4f354.png" width=300 />